index.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Alick's Portfolio</title>
+  <meta name="Alick Nyirenda" content="Alick Portfolio">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <p class="header-text">Alick Nyirenda </p>
+    <a href="#"><i class="header-icon fa fa-bars fa-lg"></i></a>
+  </header>
```

Comment:
- [ ] The logo should be on the left and should be in a link (`a` tag). Also, use a header `hx` tag for the logo since you are using plain text.
- [ ] The hamburger button on the right should be wrapped in a `nav` tag. 
This orders the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group, this helps with proper accessibility, webpage optimisation and overall site aesthetics.